index.html

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
+  <h1>Hello Microverse</h1>
+</html>
```

Comment:
-[ ] [OPTIONAL] You are are missing the `<html>` ,` <head>` , `<title>` and `<body>` tags as well as `<meta>` tags for your project. These tags and structure are useful for the readability of your project as it gets bigger, as well as to help  browsers understand how to display content or reload page, and also to improve SEO.

You can check this[ article on W3 schools](https://www.w3schools.com/html/html_head.asp) , with explanation and examples of the tags